src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -834,6 +834,10 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->booleanNode('disable_not_compromised_password')
+                            ->defaultFalse()
```

Comment:
I suppose that you are going to submit a PR on symfony/recipes to adapt the value for the `test` env?

Reply:
Yes :)